src/main/java/org/commcare/formplayer/beans/menus/ErrorBean.java

```diff
@@ -1,12 +1,21 @@
 package org.commcare.formplayer.beans.menus;
 
+import java.util.Objects;
```

Comment:
changes to this file are to allow easier usage in tests (new constructor + equality methods)